grpc/src/main/java/com/linecorp/armeria/server/grpc/HandlerRegistry.java

```diff
@@ -150,6 +161,14 @@ boolean needToUseBlockingTaskExecutor(ServerMethodDefinition<?, ?> methodDef) {
         return blockingMethods.contains(methodDef);
     }
 
+    @Nullable
+    GrpcExceptionHandlerFunction getExceptionHandler(ServerMethodDefinition<?, ?> methodDef) {
+        if (!grpcExceptionHandlers.containsKey(methodDef)) {
+            return defaultExceptionHandler;
+        }
+        return grpcExceptionHandlers.get(methodDef);
```

Comment:
`defaultExceptionHandler` is generated based on the mapping between exception and GrpcStatus set in `GrpcServiceBuilder`, and is used to distinguish it from the one created by `GrpcExceptionHandler` annotation based.

> Set DefaultGrpcExceptionHandler as a fallback exception handler to all exception handlers in grpcExceptionHandlers so that they never return null.

This feature has been already provided by the following implementation, WDYT?
https://github.com/line/armeria/blob/main/grpc/src/main/java/com/linecorp/armeria/internal/common/grpc/GrpcStatus.java#L97-L144